packages/safe-tools-client/app/components/setup-safe-modal/index.gts

```diff
@@ -39,44 +54,51 @@ export default class SetupSafeModal extends Component<Signature> {
             <li>You are the owner of the safe. Only the safe owner can schedule
               payments.
             </li>
-            <li>Once you have scheduled payments, the module (source code here)  {{! TODO: Add link }}
+            <li>Once you have scheduled payments, the module
+              <a
+                href='https://github.com/cardstack/cardstack-module-scheduled-payment'
+                target='_blank'
+                rel='external'
+              >
+                (source code here)
+              </a>
               triggers the payments at the appointed time.
             </li>
             <li>Creating the safe and enabling the module requires a one-time gas
-              fee.
+               fee.
             </li>
           </ul>
-          {{! TODO: Gas cost should be handled on this component or be a param ? }}
-          <b>Estimated gas cost: {{gasCost}}</b>
-          {{#if hasEnoughGas}}
-            <div class='safe-setup-modal__section-wallet-info'>
-              {{svgJar
-                'icon-check-circle-ht'
-                class='safe-setup-modal__section-icon'
-              }}
-              <p>
-                Your wallet has sufficient funds to cover the estimated gas cost.
-              </p>
-            </div>
-          {{/if}}
-          {{! TODO: What's the behavior for not having enough gas ? }}
+          <b>Estimated gas cost: {{this.gasCost}}</b>
+          <div class='safe-setup-modal__section-wallet-info'>
+            {{svgJar
+              (if this.notEnoughBalance 'icon-x-circle-ht' 'icon-check-circle-ht')
+              class='safe-setup-modal__section-icon'
+              style=(cssVar
+                icon-color=(if this.notEnoughBalance 'var(--boxel-red)' 'var(--boxel-green)')
+              )
+            }}
+            <p>
+              Your wallet has {{if this.notEnoughBalance 'in'}}sufficient funds to cover the estimated gas cost.
```

Comment:
We’re in a bit of a strange state as we get started with this application as we don’t really have tests, but once conditionals start to appear that’s when I usually want to see them. But since there’s no way to make `notEnoughBalance` be true programatically yet, we’ll just need to remember to write tests for this when there’s sufficient test infrastructure to mock wallet characteristics 🤓